test/manual/api/structure_signature_api_tests.sh

```
@@ -0,0 +1,462 @@
+    name: "Wormhole K162",
+    description: "Leads to unknown space",
+    type: "Wormhole",
+    linked_system_id: 30000144,
+    kind: "cosmic_signature",
+    group: "wormhole",
+    custom_info: "Fresh",
+    solar_system_id: ($sid|tonumber),
+    solar_system_name: $name
+  }')
+  echo "DEBUG: payload=$payload"
+  local raw=$(make_request POST "$API_BASE_URL/api/maps/$MAP_SLUG/signatures" "$payload")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)